starport/templates/app/templates/Makefile.plush

```diff
@@ -0,0 +1,7 @@
+all: build
```

Comment:
No sure if we should introduce a `Makefile` again. We actually intentionally removed it before to reduce one more dependency (make). :grin: I think we can also think about adding a `build` cmd to Starport as discussed in today's DevX call. cc @fadeev

Reply:
Yes, `starport build` is something we definitely need. @ilgooz is this something you think you could do before the `v0.11.0? Initially, it can be as simple as the contents of this Makefile.

Reply:
I think we can merge this PR as is and then when we have `build`, we'll remove the Makefile.

Reply:
heh :) 

I didn't know that I was adding something that'd already been removed.

Pro-Makefile:

* I figure that people will end up making Makefiles on their own anyhow
* Without it, I figure we would end up with build scripts, for example in CI
* gives clarity on next steps after starport serve

Anti-Makefile:

* The dependency thing is real.  On arch/ubuntu you need base-devel or build-essential to get Make.

Proposed solution:

Makefile and build tool.

Reply:
> Yes, `starport build` is something we definitely need. @ilgooz is this something you think you could do before the `v0.11.0? Initially, it can be as simple as the contents of this Makefile.

definitely.